lambda/matching.ml

```diff
@@ -3100,6 +3257,50 @@ let rec compile_match repr partial ctx (m : pattern_matching) =
       (bind_check str v arg lam, total)
   | _ -> assert false
 
+and compile_simplified repr partial ctx (m : Simple.clause pattern_matching) =
+  match m with
+  | { cases = []; args = [] } -> comp_exit ctx m
+  | { args = ((Lvar v as arg), str) :: argl } ->
+      (* Not sure we need to do anything with arg/v *)
```

Comment:
I am not sure what this comment is referring too? Is it a point above?

Reply:
We removed the unclear comment.